src/cpp/session/modules/SessionProfiler.R

```diff
@@ -174,6 +179,11 @@
          unlink(profileDir, recursive = TRUE)
       }
 
+      rsconnectDir <- file.path(resources$tempPath, "rsconnect", "documents", paste(filePrefix, ".html", sep = ""))
+      if (dir.exists(rsconnectDir)) {
```

Comment:
Changed! I'm testing now with `2.11.1`. However, came to realize that a bunch of rprof-related dependencies require higher R versions. But I believe it's still good to lower the bar.